boards/arm/b_l072z_lrwan1/doc/b_l072z_lrwan1.rst

```diff
@@ -0,0 +1,233 @@
+        - Bullet-proof front end: IIP3 = -12.5 dBm
+        - 89 dB blocking immunity
+        - Low Rx current of 10 mA, 200 nA register retention
+        - Fully integrated synthesizer with a resolution of 61 Hz
+        - Built-in bit synchronizer for clock recovery
+        - Sync word recognition
+        - Preamble detection
+        - 127 dB+ dynamic range RSSI
+
+- SMA and U.FL RF interface connectors
+- Including 50 ohm SMA RF antenna
+- On-board ST-LINK/V2-1 supporting USB re-enumeration capability
+
+- USB ST-LINK functions:
+- Board power supply:
+
+        - Through USB bus or external VIN/3.3 V supply voltage or batteries
+- 3xAAA-type-battery holder for standalone operation
+- 7 LEDs:
+
+        - 4 general-purpose LEDs
+        - A 5 V-power LED
+        - An ST-LINK-communication LED
+        - A fault-power LED
+        - 2 push-buttons (user and reset)
+- Arduino* Uno V3 connectors
+
+.. image:: img/b-l072z-lrwan1.png
+     :width: 500px
+     :align: center
+     :height: 325px
+     :alt: B-L072Z-LRWAN1
+
+More information about the board can be found at the `B-L072Z-LRWAN1 website`_.
+
+Hardware
+********
+
+The STM32L072CZ SoC provides the following hardware IPs:
+
+- Ultra-low-power (down to 0.29 µA Standby mode and 93 uA/MHz run mode)
+- Core: ARM* 32-bit Cortex*-M0+ CPU, frequency up to 32 MHz
+- Clock Sources:
+
+        - 1 to 32 MHz crystal oscillator
+        - 32 kHz crystal oscillator for RTC (LSE)
+        - Internal 16 MHz factory-trimmed RC ( |plusminus| 1%)
+        - Internal low-power 37 kHz RC ( |plusminus| 5%)
+        - Internal multispeed low-power 65 kHz to 4.2 MHz RC
+- RTC with HW calendar, alarms and calibration
+- Up to 24 capacitive sensing channels: support touchkey, linear and rotary touch sensors
+- 11x timers:
+
+        - 2x 16-bit with up to 4 channels
+        - 2x 16-bit with up to 2 channels
+        - 1x 16-bit ultra-low-power timer
+        - 1x SysTick
+        - 1x RTC
+        - 2x 16-bit basic for DAC
+        - 2x watchdogs (independent/window)
+- Up to 84 fast I/Os, most 5 V-tolerant.
+- Memories
+
+        - Up to 192 KB Flash, 2 banks read-while-write, proprietary code readout protection
+        - Up to 20 KB of SRAM
+        - External memory interface for static memories supporting SRAM, PSRAM, NOR and NAND memories
+- Rich analog peripherals (independent supply)
+
+        - 1x 12-bit ADC 1.14 MSPS
+        - 2x 12-bit DAC
+        - 2x ultra-low-power comparators
+- 11x communication interfaces
+
+        - USB OTG 2.0 full-speed, LPM and BCD
+        - 3x I2C FM+(1 Mbit/s), SMBus/PMBus
+        - 4x USARTs (ISO 7816, LIN, IrDA, modem)
+        - 6x SPIs (4x SPIs with the Quad SPI)
+- 7-channel DMA controller
+- True random number generator
+- CRC calculation unit, 96-bit unique ID
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell*
+
+
+More information about STM32L072CZ can be found here:
+
+        - `STM32L072CZ on www.st.com`_
```

Comment:
This renders better when the bullet points are not indented:

    More information about STM32L072CZ can be found at:

    - `STM32L072CZ on www.st.com`_
    - `STM32L0x2 reference manual`_